projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx

```diff
@@ -114,27 +119,113 @@ type AiAssistantControlComponentProps = {
 	 * Whether the dropdown is disabled.
 	 */
 	disabled?: boolean;
-
-	onChange: ( item: PromptTypeProp, options?: AiAssistantDropdownOnChangeOptionsArgProps ) => void;
-
-	onReplace: () => void;
 };
 
 export default function AiAssistantDropdown( {
 	key,
 	label,
+	blockType,
 	exclude = [],
 	requestingState,
```

Comment:
What is `requestingState`?

Reply:
`requestState` represents the different states that a suggestion request could have: `init`, `requesting`, `suggesting`, and `done`. It isn't used anymore. Let's remove it.